exchanges/okx/okx.go

```diff
@@ -54,306 +60,14 @@ const (
-	errNoCandlestickDataFound                  = errors.New("no candlesticks data found")
-	errInvalidWebsocketEvent                   = errors.New("invalid websocket event")
-	errMissingValidChannelInformation          = errors.New("missing channel information")
-	errNilArgument                             = errors.New("nil argument is not acceptable")
-	errNoOrderParameterPassed                  = errors.New("no order parameter was passed")
-	errMaxRfqOrdersToCancel                    = errors.New("no more than 100 Rfq cancel order parameter is allowed")
-	errMalformedData                           = errors.New("malformed data")
-	errInvalidUnderlying                       = errors.New("invalid underlying")
-	errMissingRequiredParameter                = errors.New("missing required parameter")
-	errMissingMakerInstrumentSettings          = errors.New("missing maker instrument settings")
-	errInvalidSubAccountName                   = errors.New("invalid sub-account name")
-	errInvalidAPIKey                           = errors.New("invalid api key")
-	errInvalidAlgoID                           = errors.New("invalid algo id")
-	errInvalidMarginTypeAdjust                 = errors.New("invalid margin type adjust, only 'add' and 'reduce' are allowed")
-	errInvalidAlgoOrderType                    = errors.New("invalid algo order type")
-	errEmptyArgument                           = errors.New("empty argument")
-	errInvalidIPAddress                        = errors.New("invalid ip address")
-	errInvalidAPIKeyPermission                 = errors.New("invalid API Key permission")
-	errInvalidResponseParam                    = errors.New("invalid response parameter, response must be non-nil pointer")
-	errEmptyPlaceOrderResponse                 = errors.New("empty place order response")
-	errTooManyArgument                         = errors.New("too many cancel request params")
-	errIncompleteCurrencyPair                  = errors.New("incomplete currency pair")
-	errInvalidDuration                         = errors.New("invalid grid contract duration, only '7D', '30D', and '180D' are allowed")
-	errInvalidProtocolType                     = errors.New("invalid protocol type, only 'staking' and 'defi' allowed")
-	errExceedLimit                             = errors.New("limit exceeded")
-	errOnlyThreeMonthsSupported                = errors.New("only three months of trade data retrieval supported")
-	errOnlyOneResponseExpected                 = errors.New("one response item expected")
-	errNoInstrumentFound                       = errors.New("no instrument found")
 )
 
 /************************************ MarketData Endpoints *************************************************/
 
 // OrderTypeFromString returns order.Type instance from string
 func (ok *Okx) OrderTypeFromString(orderType string) (order.Type, error) {
-	switch strings.ToLower(orderType) {
+	orderType = strings.ToLower(orderType)
+	switch orderType {
 	case OkxOrderMarket:
```

Comment:
Not changes done in this PR, but we can rid the Okx prefix stutter for these internal order types and same goes for the instrument types